5_databases/week2/app.js

```diff
@@ -0,0 +1,39 @@
+import express from "express";
+import { readFileSync } from "node:fs";
+
+const app = express();
+const port = process.env.PORT || 3003;
+
+
+const data = readFileSync("./documents.json", (err, data) => {
```

Comment:
this is a legitimate right way to read documents.json. here is a hack for you. if you change documents.json to a js file and define the documents variable in that file you can just import it in this file :)